--- a/setup.py	2026-06-10 21:28:43.339437191 -0500
+++ b/setup.py	2026-06-10 21:28:56.019588153 -0500
@@ -2,7 +2,6 @@
 # type: ignore
 import pathlib
 
-import pkg_resources
 from setuptools import find_namespace_packages, setup
 
 from build_helpers.build_helpers import (
@@ -16,8 +15,9 @@
 
 with pathlib.Path("requirements/requirements.txt").open() as requirements_txt:
     install_requires = [
-        str(requirement)
-        for requirement in pkg_resources.parse_requirements(requirements_txt)
+        line
+        for line in (line.strip() for line in requirements_txt)
+        if line and not line.startswith("#") and not line.startswith("-")
     ]
 
 
