--- a/setup.py	2026-06-10 21:29:06.806883112 -0500
+++ b/setup.py	2026-06-10 21:29:17.112041811 -0500
@@ -10,7 +10,6 @@
 """
 import pathlib
 
-import pkg_resources
 import setuptools
 
 from build_helpers.build_helpers import (
@@ -24,8 +23,9 @@
 
 with pathlib.Path("requirements/base.txt").open() as requirements_txt:
     install_requires = [
-        str(requirement)
-        for requirement in pkg_resources.parse_requirements(requirements_txt)
+        line.strip()
+        for line in requirements_txt
+        if line.strip() and not line.strip().startswith(("#", "-"))
     ]
 
 
