--- a/setup.py	2026-06-10 21:29:06.809834177 -0500
+++ b/setup.py	2026-06-10 21:29:20.350111254 -0500
@@ -1,8 +1,8 @@
 import setuptools
-from pkg_resources import get_distribution
+from importlib.metadata import version
 
 try:
-    get_distribution("setuptools>=39.2.0")
+    assert tuple(int(x) for x in version("setuptools").split(".")[:2]) >= (39, 2)
 except Exception as e:
     raise AssertionError(
         "Please upgrade setuptools by `pip install -U setuptools`: {}".format(
